src/WebJobs.Script/Description/Node/NodeFunctionInvoker.cs

```diff
@@ -590,11 +591,27 @@ private static void Initialize()
                 }
                 return Task.FromResult<object>(null);
             });
+
             var context = new Dictionary<string, object>()
             {
                 { "handleUncaughtException", handle }
             };
 
+            if (Host.ScriptConfig.HostConfig?.Tracing?.ConsoleLevel == TraceLevel.Verbose)
+            {
+                context.Add("console", (Func<object, Task<object>>)(msg =>
+                {
+                    try
+                    {
+                        Host.TraceWriter.Verbose(msg as string, PrimaryHostUserTraceProperties);
```

Comment:
You should be able to log this with ILogger using `Host.Logger?.LogDebug(msg)`. Nothing from the ILogger gets to our System logs (that's all done by a TraceWriter), so you don't have to worry about that. 

That log would be created with a `Host.General` category, which may not be what you want for filtering purposes here. Creating a new category could be beneficial. If that's the case:
`ILogger logger = Host.ScriptConfig.HostConfig.LoggerFactory.CreateLogger("Host.NodeFunctionInvoker");` 

The benefit to that is that someone would be able to explicitly enable debug logging for this stuff only. 

If you're still selectively wrapping this, you'd make it something like: 

`if (Host.ScriptConfig.HostConfig?.Tracing?.ConsoleLevel == TraceLevel.Verbose ||
    Host.ScriptConfig.LogFilter.CategoryLevels["Host.NodeFunctionInvoker"] > LogLevel.Debug)`